src/connection.rs

```diff
@@ -22,7 +25,7 @@ static DEFAULT_PORT: u16 = 6379;
 pub fn parse_redis_url(input: &str) -> Result<url::Url, ()> {
     match url::Url::parse(input) {
         Ok(result) => match result.scheme() {
-            "redis" | "redis+unix" | "unix" => Ok(result),
+            "redis" | "redis+tls" | "redis+unix" | "unix" => Ok(result),
```

Comment:
I remember that some clients did use `rediss:` for secure connections (see also [the iana spec](https://www.iana.org/assignments/uri-schemes/prov/redis)).

Did you look what other clients are currently using?
(I'm ok with `redis+tls`, but if `rediss:` is used widely we maybe should have that as well)

Reply:
I merely copied `redis+unix` for consistency, but I guess the IANA spec is a much better source of truth.